compiler/src/iree/compiler/Codegen/SPIRV/SPIRVVectorizeLoadStore.cpp

```diff
@@ -292,7 +301,29 @@ class ProcessFunctionArgument final
 
   LogicalResult
   matchAndRewrite(func::FuncOp funcOp, OpAdaptor adaptor,
-                  ConversionPatternRewriter &rewriter) const override;
+                  ConversionPatternRewriter &rewriter) const override {
+    TypeConverter::SignatureConversion signatureConverter(
+        funcOp.getFunctionType().getNumInputs());
+    for (const auto &[index, arg] : llvm::enumerate(funcOp.getArguments())) {
```

Comment:
No need for `const &` over enumerate -- it makes it seem like you can't modify `arg` but actually you can. And it's more typing.